src/phy/mod.rs

```diff
@@ -243,6 +275,8 @@ pub struct DeviceCapabilities {
     /// If the network device is capable of verifying or computing checksums for some protocols,
     /// it can request that the stack not do so in software to improve performance.
     pub checksum: ChecksumCapabilities,
+
+    pub hardware_timestamping: bool,
```

Comment:
What is the `hardware_timestamping` field used for?

Reply:
Oops, that's a bit of a leftover from experimenting. If `smoltcp` ever gets "built-in" (through something like a `PtpSocket`) support for timestamping, allowing `Device`s to indicate that they have support for hardware timestamping is probably desirable. Not relevant for this PR though, so I'll remove this field.